### DISCUSSION TITLE
Minimum angular velocity  /kachaka/manual_control/cmd_vel

### DISCUSSION THREAD

**Discussion by LonelyPaprika** [pf-robotics/kachaka-api#71] [General]
I observe that when I send a low angular velocity (0.01 rad/s), the angular velocity of the robot is actually around 0.15 rad/s. I observe this thanks to the kachaka/odometry. 

Are you setting a minimum velocity threshold ? 

Thank you.

**Comment by terakoji-pfr** [comment]
KACHAKA cannot control velocity below 0.01 [rad/s] due to odometry limitations; specify a velocity of 0.02 [rad/s] or greater.
